### ISSUE TITLE
Errors when using Friendly Schema names

### ISSUE THREAD

**Issue by DeanGross** [MicrosoftDocs/Azure-RMSDocs#1036]
I am getting Failed to resolve scalar expression for the following functions: Access, AccessDenied, and InformationTypesMatches. Has something been changed?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0b75d85e-3cea-279c-feaf-c8f828f3d201
* Version Independent ID: 97d9a1b1-4db3-a1c0-69fe-8032ce615fc7
* Content: [Central reporting for Azure Information Protection](https://docs.microsoft.com/en-us/azure/information-protection/reports-aip#configure-a-log-analytics-workspace-for-the-reports)
* Content Source: [Azure-RMSDocs/reports-aip.md](https://github.com/Microsoft/Azure-RMSDocs/blob/master/Azure-RMSDocs/reports-aip.md)
* Service: **information-protection**
* Sub-service: **analytics**
* GitHub Login: @cabailey
* Microsoft Alias: **cabailey**

**Comment by cabailey** [comment]
@DeanGross Nothing has changed to my knowledge, but I am checking.  Was this working for you previously?  I see you've asked on Yammer as well.

**Comment by cabailey** [comment]
I have new info that might explain this - I've just been told that these columns are created only when data for them is received.  Do you see data for them in **Azure Information Protection - Activity logs**?  For example, "Access" in the Activity column as a result of opening a protected document on a Windows computer?

**Comment by DeanGross** [comment]
Yes, I do the Access and Access Denied Activity events in the AIP Activity Logs (Preview) pagem however, I don't see the InformationTypesMatches

**Comment by DeanGross** [comment]
I don't know if if ever worked, I 'm trying this for the first time

**Comment by cabailey** [comment]
OK, still looking into this.  But if this is urgent, please open a ticket with Support.

**Comment by cabailey** [comment]
@DeanGross  Apologies for the delay (Jewish holidays)

**Comment by cabailey** [comment]
@DeanGross  Thank you for your patience while we unravel this.  It seems Access and AccessDenied were not implemented as event functions, only as potential values in the Activity field.  I've now removed these from the table.  Still investigating InformationTypesMatches.

**Comment by cabailey** [comment]
@DeanGross  I finally have a confirmed answer for you: InformationTypesMatches isn't implemented either.  Instead, the values are included in DiscoveredInformationTypes.

I've removed InformationTypesMatches from the table.  There was nothing to change in the description for DiscoveredInformationTypes.
